<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cClientView</class>
 <widget class="QGroupBox" name="cClientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>84</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>client:</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="userLabel">
         <property name="text">
          <string>user:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userLineEdit">
         <property name="text">
          <string>darkjavi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="text">
          <string>largoygrueso</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="hostLabel">
         <property name="text">
          <string>host:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hostLineEdit">
         <property name="text">
          <string>tesla</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portLineEdit">
         <property name="text">
          <string>31415</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="connectionStatuslabel">
     <property name="text">
      <string>Not connected...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
